docs/sections/user_guide/cli/drivers/esg_grid.rst

```diff
@@ -0,0 +1,97 @@
+``esg_grid``
+============
+
+The ``uw`` mode for configuring and running the :ufs-utils:`regional_esg_grid<regional-esg-grid>` component.
```

Comment:
I'm working on this link for the `esg_grid`. The naming convention we went with for this driver is `esg_grid`, and the UFS Doucmentation for the driver is called under `regional_esg_grid` causing problems with linking. I haven't been able to figure this one out and it's causing a broken link if I use `esg_grid` here instead of `regional_esg_grid`, but after looking at the documentation it still doesn't link correctly.

The broken link address: https://noaa-emcufs-utils.readthedocs.io/en/latest/ufs_utils.html#esg-grid
The correct link address: https://noaa-emcufs-utils.readthedocs.io/en/latest/ufs_utils.html#regional-esg-grid

Reply:
When I looked earlier they still weren't working properly/ Seems like they are now. Possibly just had to wait for them to build again.